### ISSUE TITLE
Sixel graphics are not drawn on xlib 1bpp and --with-imagelib=no

### ISSUE THREAD

**Issue by tsutsui** [arakiken/mlterm#93]
It looks xlib mlterm calls `ui_imagelib_load_file()` to draw sixel graphics data:
https://github.com/arakiken/mlterm/blob/f0d1c4237a38caa124675b74d90eac9ac54919ad/uitoolkit/xlib/ui_imagelib.c#L1714-L1729

but `load_sixel()` doesn't work on depth<8 so `load_file()` is called later:
https://github.com/arakiken/mlterm/blob/f0d1c4237a38caa124675b74d90eac9ac54919ad/uitoolkit/xlib/ui_imagelib.c#L431

`load_file()` calls `exec_mlimgloader()` with `pixbuf` arg, but the function in tool/mlimgloader/none.c doesn't support pixbuf so `mlimgloader` exits with an error.

**Comment by arakiken** [comment]
Thanks.
Does this fix improve it?
https://github.com/arakiken/mlterm/commit/63bb60ee8b115bdca3e775dfce612e29a1e7ecfa

**Comment by tsutsui** [comment]
I'll try it (on vax), but maybe one more depth check should be removed?
```diff
diff --git a/uitoolkit/xlib/ui_imagelib.c b/uitoolkit/xlib/ui_imagelib.c
index 13a4d5e8..9e53cff2 100644
--- a/uitoolkit/xlib/ui_imagelib.c
+++ b/uitoolkit/xlib/ui_imagelib.c
@@ -428,7 +428,7 @@ static int load_sixel(ui_display_t *disp, char *path, Pixmap *pixmap,
   GC mask_gc;
   int num_cells;
 
-  if (disp->depth < 8 || !(data = in = load_sixel_from_file(path, &w, &h, transparent))) {
+  if (!(data = in = load_sixel_from_file(path, &w, &h, transparent))) {
     return 0;
   }
 
```

**Comment by arakiken** [comment]
Thanks. https://github.com/arakiken/mlterm/commit/a4baddc173046ac0e1eaed0f1912fbb2578d769a

**Comment by tsutsui** [comment]
Confirmed fix with pkgsrc mlterm 3.9.3 + https://github.com/arakiken/mlterm/commit/63bb60ee8b115bdca3e775dfce612e29a1e7ecfa + https://github.com/arakiken/mlterm/commit/a4baddc173046ac0e1eaed0f1912fbb2578d769a

Thanks,
![mlterm-xlib-1bpp-sixel-1](https://github.com/arakiken/mlterm/assets/3054480/71354ed6-2d56-4084-a01b-1baccdb9eb7a)
![mlterm-xlib-1bpp-sixel-2](https://github.com/arakiken/mlterm/assets/3054480/5647c2b0-08f8-4119-ba91-a8f0f477b7d8)
